giraffe/src/components/AnnotationLine.tsx

```diff
@@ -120,6 +136,9 @@ export const AnnotationLine: FunctionComponent<AnnotationLineProps> = props => {
   const makeRangeRectangle = () => {
     const pixelMargin = 1
 
+    // cannot just directly reference the handleMouseEnter/Leave functions;
+    // that doesn't work (it gets called immediately).
+    // so need the arrow function when using createElement
```

Comment:
Only if you call `handleMouseEnter` immediately. If you don't call the function immediately, it doesn't get called immediately.

Reply:
well, when I just put only the function name it errored out.  using the fat arrows fixed it.

I tried to find documentation on createElement but could not find it.

Reply:
`onMouseEnter: handleMouseLeave,` works, same for Leave

were you referring to `handleMouseLeave,` ? there's no property called `handleMouseLeave` on the object passed to createElement